frame/clearing-house/src/lib.rs

```diff
@@ -2308,8 +2306,7 @@ pub mod pallet {
 				};
 				let last_oracle_10bp =
 					last_oracle.try_div(&T::Decimal::saturating_from_integer(1000))?;
-				oracle = math::clip(
-					oracle,
+				oracle = oracle.clamp(
```

Comment:
same comment from PR #1193: I think `oracle_price` is more readable than `oracle`

Reply:
That's on me. I'll change it in the Clearing House pallet when I open a new PR

Reply:
@kevin-composable, regarding your question about where the function `clamp` is declared, [it is in the standard library, to every type that defines `Ord`.](https://doc.rust-lang.org/nightly/core/cmp/trait.Ord.html#method.clamp)

Reply:
I removed `math::clip` and opted to just use `clamp` because in that way we don't duplicate existing code. There are two points to consider, though: 
- In the latest version of this code block that is in `escher` one typo/bug is fixed (using `.try_sub()` for both parameters)
- This function may panic if `min > max`, so it's important to ensure that it would never happen. (currently not checked)

I can make this change in #1193 if you want, but I think it would be more suitable to @0xangelo in his next PR.

The reasoning for me to change the method in this code block is because I was moving some helper functions and removing unneeded code.

Reply:
I already renamed `oracle` to `oracle_price` in my local branch, so you don't need to worry about that

Reply:
Just need to keep in mind that using `clamp` rather than `math::clip` might panic. 
Would you prefer to re-add the `math::clip` helper? 
Maybe adding it as a trait method rather than a helper function would be better. I can do it in a separate PR if you wish.
cc/ @0xangelo

Reply:
I'll add a `TryClamp` trait in my next PR, sounds good?

Reply:
That is, after you merge the math helpers PR #1193

Reply:
Done in #1284